MatrixMultiplication/MatrixMultiplication/Statistics.cs

```diff
@@ -0,0 +1,124 @@
+using System.Diagnostics;
+using Aspose.Pdf;
+
+namespace MatrixMultiplication;
+
+/// <summary>
+/// A static class that allows you to generate a set of reports  for matrices with different sizes.
+/// </summary>
+internal static class Statistics
+{
+    private static readonly Random Rand = new();
```

Comment:
```suggestion
    private static readonly Random rand = new();
```
Кстати, не сокращайте имена никогда